monitoring.yml

```diff
@@ -218,26 +217,7 @@ instance_groups:
         # changing this retention policy has no effect due a bug in the upstream boshrelease
         # you must change the retention policy by hand by running the following command on the influxdb node
         # curl -X POST -G 'http://localhost:8086/query?pretty=true' --data-urlencode "q=ALTER RETENTION POLICY \"default\" ON \"production\" DURATION :duration:;"
-        retention: 90d
```

Comment:
I think we should probably leave this in and set it to 30d to have a documented record of what it's supposed to be